internal/cmd/flink/command_statement_exceptions_list.go

```diff
@@ -0,0 +1,55 @@
+package flink
+
+import (
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/spf13/cobra"
+)
+
+func (c *command) newStatementExceptionsListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "list <statement name>",
+		Short:             "Get exceptions for a Flink SQL statement.",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validStatementArgs),
+		RunE:              c.exceptionsList,
+	}
+
+	pcmd.AddForceFlag(cmd)
```

Comment:
Isn't that only used for delete statements?

Reply:
good catch, removed